### DISCUSSION TITLE
我的 2021 盘点 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#257] [Announcements]
# 我的 2021 盘点 — 码志

对于 2021 年，如果仅从岁月流逝的角度，当然是期望时光时光慢些吧；如果从这一年外界的疯狂变幻来讲，又想对它说慢走不送。但终会无力地看清，时间的洪流上并无刻度，你喜欢或者不喜欢，它并不在意，兀自「逝者如斯夫，不舍昼夜」。

https://mazhuang.org/2022/01/01/my-2021/

**Comment by wenbingyin** [comment]
测试评论
